app/src/test/java/com/simplemobiletools/calculator/MainActivityTest.kt

```diff
@@ -3,6 +3,7 @@ package com.simplemobiletools.calculator
 import com.simplemobiletools.calculator.activities.MainActivity
 import com.simplemobiletools.calculator.helpers.*
 import junit.framework.Assert.assertEquals
+import kotlinx.android.synthetic.main.activity_main.*
```

Comment:
what is this import doing?

Reply:
we need it to call things like `activity.btn_clear.text`